src/halmos/warnings.py

```diff
@@ -5,6 +5,7 @@
 from .utils import color_warn
 
 logger = logging.getLogger(__name__)
+logging.basicConfig()
```

Comment:
karma, this is needed for the basic formatting of logging messages with the prefix of "WARNING:halmos.warnings:".
so far, it was implicitly executed when importing crytic_compile. please let me know if you don't want to keep this.

Reply:
ah I see! no it's great, let's keep it